src/scss/pages/_profile.scss

```diff
@@ -0,0 +1,201 @@
+      }
+
+      .profile-user {
+        display: flex;
+        flex-direction: column;
+        justify-content: start;
+        align-items: start;
+        width: 100%;
+        height: 100%;
+
+        .profile-user-nick {
+          text-align: center;
+        }
+
+        .profile-rating {
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+          align-items: start;
+          gap: 10px;
+          height: 100%;
+        }
+
+        .profile-actions {
+          height: 100%;
+          display: flex;
+          flex-direction: row;
+          margin-top: 10px;
+          justify-content: end;
+          gap: 10px;
+          //   @media (max-width: 1500px) {
+          //     flex-direction: row;
+          //     margin-top: 10px;
+          //   }
+        }
+      }
+    }
+
+    .profile-offers-smaller-screen {
+      display: none;
+      @media (max-width: 780px) {
+        display: flex;
+        flex-direction: column;
+        gap: 20px;
+        background-color: v.$blue-dark;
+        padding-bottom: 20px;
+        margin-bottom: 20px;
+      }
+
+      .profile-client-data-container-smaller-screen {
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        margin-top: 20px;
+      }
+
+      .profile-offer-card-smaller-screen {
+        display: flex;
+        justify-content: center;
+      }
+
+      .profile-no-offers {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        height: 100%;
+        gap: 20px;
+
+        .profile-no-offers-add-offer-button {
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+          align-items: center;
+          width: 100%;
+          margin-top: 10px;
+        }
+      }
+    }
+
+    .profile-add-a-comment {
+      margin-top: 10px;
+      margin-bottom: 20px;
+    }
+
+    .profile-comments-page {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: end;
+      gap: 10px;
+
+      .profile-comments-page-buttons {
+        display: flex;
+        justify-content: center;
+        gap: 10px;
+      }
+    }
+  }
+
+  .profile-right {
+    display: flex;
+    max-height: 100vh;
+    flex: 5;
+    overflow-x: auto;
+    flex-direction: column;
+    color: v.$white;
+
+    @media (max-width: 780px) {
+      display: none;
+    }
+
+    .profile-client-data-container {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      .profile-client-card {
+        margin: 10px;
+        width: 50%;
+      }
+    }
+
+    .profile-right-offers {
+      overflow-x: auto;
+      display: flex;
+      flex-direction: row;
+      justify-content: start;
+      padding: 20px;
+      height: 100%;
+
+      gap: 20px;
+
+      /* hide scrollbar */
+      scrollbar-width: none; /* Firefox */
```

Comment:
Jest jakiś problem w przypadku przeglądarki Firefoxa, że jest to potrzebne? 
Bo u mnie na Chromie, na komputerze, ta właściwość powoduje, że nie pokazuje mi się poziomy scrollbar przy ofertach gdy jest overflow-x.

Reply:
Chciałem schować ten scrollbar, wtedy można przytrzymać shift i scrollować w poziomie, ale rzeczywiście lepiej to zmienić, bo nie jest to user friendly